next-app/src/app.html

```diff
@@ -1,13 +1,11 @@
 <!DOCTYPE html>
-<html lang=en>
-	<head>
-		<meta charset=utf-8 />
-		<meta name=viewport content='width=device-width initial-scale=1' />
-		<link rel=icon href=%sveltekit.assets%/favicon.ico />
-
-		%sveltekit.head%
-	</head>
-	<body>
-		%sveltekit.body%
-	</body>
+<html lang="en">
```

Comment:
these double-quotes seem a bit random, see the `link rel=icon` below so I'd suggest we not enforce them

Reply:
Yeah, I'm confused by this one too 🤔